docs/source/share.rst

```diff
@@ -0,0 +1,662 @@
+
+   .. code-block::
+
+        git add datasets/<my-new-dataset>
+        git commit
+        git push -u origin my-new-dataset
+
+8. Go back to your fork on Github, and click on **Pull request** to open a pull request on the main repository for review.
+
+Community dataset
+^^^^^^^^^^^^^^^^^
+
+Sharing a community dataset will require you to create an account on `hf.co <https://huggingface.co/join>`_ if you don't already have one. You can directly create a `new dataset repository <https://huggingface.co/new-dataset>`_ from your account on the Datasets Hub, but this guide will show you how to upload a dataset from the terminal.
+
+1. Make sure you are in the virtual environment where you installed Datasets, and run the following command:
+
+   .. code-block::
+
+        huggingface-cli login
+
+2. Login using your Datasets Hub credentials, and create a new dataset repository:
+
+   .. code-block::
+
+        huggingface-cli repo create your_dataset_name --type dataset
+
+    If you want to create a repository under a specific organization, add the ``-organization`` flag:
+
+   .. code-block::
+
+        huggingface-cli repo create your_dataset_name --type dataset --organization your-org-name
+
+3. Install `Git LFS <https://git-lfs.github.com/>`_ and clone your repository:
+
+   .. code-block::
+
+        # Make sure you have git-lfs installed
+        # (https://git-lfs.github.com/)
+        git lfs install
+
+        git clone https://huggingface.co/datasets/username/your_dataset_name
+
+4. Now is a good time to check your directory to ensure the only files you are uploading to the Hub are:
+
+   * ``README.md`` is a Dataset card that describes the datasets contents, creation, and usage.
+
+   * ``your_dataset_name.py`` is your dataset loading script.
+
+   * ``dataset_infos.json`` contains the metadata about the dataset.
+
+   * ``dummy_data`` holds a small subset of data from the dataset for tests and preview.
+
+   * Raw files of the dataset.
+
+5. It is important to add the large data files first with ``git lfs track`` or else you will encounter an error later when you push your files:
+
+   .. code-block::
+
+        cp /somewhere/data/*.json .
+        git lfs track *.json
+        git add .gitattributes
+        git add *.json
+        git commit -m "add json files"
+
+6. Add the dataset loading script and metadata file:
+
+   .. code-block::
+
+        cp /somewhere/data/dataset_infos.json .
+        cp /somewhere/data/load_script.py .
+        git add --all
+
+7. Verify the files have been correctly staged, then you can commit and push your files:
+
+   .. code-block::
+
+        git status
+        git commit -m "First version of the your_dataset_name dataset."
+        git push
+
+
+Congratulations, your dataset has now been uploaded to the Datasets Hub where anyone can load it with a single line of code! 🤗
+
+.. code-block::
+
+    dataset = load_dataset("namespace/your_dataset_name")
+
+Metric
+------
```

Comment:
Maybe this can be a separate page ?

Reply:
This makes sense because we're not really sharing a metric. What do you think about moving it to the Use Metrics section, except we will rename it to just Metrics. That way users can find all the how-to info related to Metrics on one page.